### PR TITLE
Update to new external object sync

### PR DESCRIPTION
**Pull request by croomes**
Updates to the latest version of the operator toolkit which will be needed for label sync.  It changes the external object sync, see example: https://github.com/darkowlzz/operator-toolkit/blob/14649a309dc6585984e6f464788e4853dda6bacb/example/controllers/external-game-sync_controller.go

### CODE REVIEW

**Review by darkowlzz — APPROVED** [review]
LGTM!

File: controllers/namespace-delete/controller.go [hunk 1]
```diff
@@ -6,7 +6,7 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
-	objectv1 "github.com/darkowlzz/operator-toolkit/controller/external-object-sync/v1"
+	syncv1 "github.com/darkowlzz/operator-toolkit/controller/sync/v1"
```
Comment (darkowlzz) [inline controllers/namespace-delete/controller.go:9]: This and other imports are not separated correctly for some reason.
I think we only separate native and external imports.

File: controllers/node-delete/controller.go [hunk 2]
```diff
@@ -6,7 +6,7 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
-	objectv1 "github.com/darkowlzz/operator-toolkit/controller/external-object-sync/v1"
+	syncv1 "github.com/darkowlzz/operator-toolkit/controller/sync/v1"
```
Comment (darkowlzz) [inline controllers/node-delete/controller.go:9]: Same weird import separation like in namespace-delete controller.

File: controllers/namespace-delete/controller.go (hunk 1 above)
Comment (croomes) [reply controllers/namespace-delete/controller.go:6]: Yes, will fix - thanks

File: controllers/node-delete/controller.go (hunk 2 above)
Comment (croomes) [reply controllers/node-delete/controller.go:6]: Yes, will fix - thanks

**Pull request merged**